tests/jballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -997,7 +997,7 @@
             </class>
             <class name="org.ballerinalang.test.services.ErrorReturnTest">
```

Comment:
https://github.com/ballerina-platform/ballerina-lang/pull/26807/files#r518952028 this comments apply here again.

Reply:
Removed.